src/illuminatio/test_orchestrator.py

```diff
@@ -301,44 +323,47 @@ def collect_results(self, api: k8s.client.CoreV1Api):
                 if api_exception.reason == "Not Found":
                     pass
                 else:
-                    raise(api_exception)
-            logger.debug("Map names: " + str([m.metadata.name for m in result_config_maps]))
-            logger.debug("Expected names: " + str(expected_result_map_names))
+                    raise api_exception
+            self.logger.debug("Map names: " + str([m.metadata.name for m in result_config_maps]))
```

Comment:
Use lazy logging

Reply:
Could you also create a new issue -> We should use https://docs.python.org/3/howto/logging.html#optimization for "expensive" methods like this and only evaluate them if we are in the actual logging level.